watson_developer_cloud/natural_language_understanding_v1.py

```diff
@@ -82,3 +82,24 @@ def analyze(self, features, text=None, url=None, html=None,
                             headers={'content-type': 'application/json'},
                             json=body,
                             accept_json=True)
+    def listModels(self):
+        """
+        Lists the custom models available for your service instance
+        :return: dict of available custom models
+        """
+        return self.request(method='GET', url='/v1/models',
+                            params={"version": self.version},
+                            accept_json=True)
+
+    def deleteModel(self, model_id=None):
```

Comment:
You indicate that model_id can be None by making it's default value none.  I'd remove the default value and the model_id check later.    The service call will error on the None and by making it required the users knows that it should be there.